DashAI/back/models/scikit_learn/numerical_wrapper_for_text.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
+from sklearn.feature_extraction.text import CountVectorizer
+
+from DashAI.back.core.schema_fields import BaseSchema, component_field, int_field
+from DashAI.back.dataloaders.classes.dashai_dataset import DashAIDataset
+from DashAI.back.models.scikit_learn.sklearn_like_model import SklearnLikeModel
+from DashAI.back.models.text_classification_model import TextClassificationModel
+
+
+class NumericalWrapperForTextSchema(BaseSchema):
+    """
+    NumericalWrapperForText is a metamodel that allows text classification using
+    tabular classifiers and a tokenizer.
+    """
+
+    tabular_classifier: component_field(
+        description="Tabular model used as the underlying model"
+        "to generate the text classifier.",
+        default="SVC",
```

Comment:
This goes in line with https://github.com/DashAISoftware/DashAI/pull/179#discussion_r1481829598
Having a default here can be really complicated. What parameters that SVC has if the user returns nothing? I dont think component fields are meant to have a default. They just need to enable the front to do the required request to the backend to obtain all the parents.
Even if this was a placeholder component, i feel it can bring some problems.

Reply:
I also think it could lead to problems in the future, i will fix it.